compiler/ztests/pushdown.yaml

```
@@ -2,53 +2,41 @@ script: |
   export ZED_LAKE=test
   zed init -q
   zed create -q -orderby ts pool-ts
-  zc -C -O "from 'pool-ts'| x=='hello' or x==1.0" | sed -e 's/pool .*/pool POOL/'
+  zc -C -O "from 'pool-ts' | x=='hello' or x==1.0" | sed -e 's/lister .*/lister/' -e 's/seqscan .*filter/seqscan filter/'
```

Nit: Simplify the sed commands in here.
```suggestion
  zc -C -O "from 'pool-ts' | x=='hello' or x==1.0" | sed -e 's/pool .*/.../'
```

---

That clobbers the seqscan filter predicate under test.

---

How about this?
```suggestion
  zc -C -O "from 'pool-ts' | x=='hello' or x==1.0" | sed -e 's/pool [^ ]*/.../'
```